python/tvm/script/ir_builder/tir/ir.py

```diff
@@ -1532,7 +1532,7 @@ def wrapped(*args, **kwargs):
 nearbyint = _op_wrapper(_tir_op.nearbyint)
 nextafter = _op_wrapper(_tir_op.nextafter)
 popcount = _op_wrapper(_tir_op.popcount)
-power = _op_wrapper(_tir_op.power)
+pow = _op_wrapper(_tir_op.power)  # pylint: disable=redefined-builtin
```

Comment:
I was thinking if we could add a `pow` method to `tvm/tir/op.py` so that it could be consistent

Reply:
Done.